driver.go

```diff
@@ -98,6 +98,27 @@ func (hd *hetznerDriver) Create(req *volume.CreateRequest) error {
 		}
 	}
 
+	uid := getOption("uid", req.Options)
+	gid := getOption("gid", req.Options)
+	if uid != "0" || gid != "" {
```

Comment:
the default value for `gid` above is "0", right? shouldn't we change this to match it?

Reply:
ouch. good catch